client/src/components/NationwideMap.vue

```diff
@@ -286,7 +286,7 @@ export default defineComponent({
      * Sets up interaction handlers for map.
      */
     setUpInteractionHandlers(): void {
-      if (this.map == null) return;
+      if (this.map == null || this.scorecard) return;
```

Comment:
I'd lke to rename this like "basicMap" instead of scorecard bc this component should not have to know about the pages it is served on

Reply:
that makes sense! do you want to do separately? Feel like should be small enough to do in this PR.

Reply:
Updated!